### PR TITLE
Vendor and package cluster capacity in origin.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@derekwaynecarr @eparis @sdodson @tdawson

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@stevekuznetsov @smarterclayton

File: images/cluster-capacity/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
+# This is the cluster capacity tool.
+#
+# The standard name for this image is openshift/origin-cluster-capacity
+#
+FROM centos:centos7
+
+COPY bin/hypercc /bin/hypercc
```
Comment (stevekuznetsov) [inline images/cluster-capacity/Dockerfile]: Would be superb if we installed from RPM here.

Comment (aveshagarwal) [inline images/cluster-capacity/Dockerfile]: how would that work who wanted to build images from local sources like OS_RELEASE=n ./hack/build-images.sh?

Comment (stevekuznetsov) [inline images/cluster-capacity/Dockerfile]: Would use same mechanism as #13642 -- can be a follow-up, depending on which lands first.

**Pull request closed without merging**